rgw/v2/tests/curl/test_cors_using_curl.py

```diff
@@ -0,0 +1,166 @@
+"""
+Usage: test_cors_using_curl.py -c <input_yaml>
+polarion: CEPH-10355
+<input_yaml>
+    Note: Following yaml can be used
+    test_cors_using_curl.yaml
+    
+Operation:
+    Test CORS on a bucket using CURL API calls.
```

Comment:
@TejasC88 Request to make the operation more descriptive step by step would be better